.github/workflows/checking.yml

```diff
@@ -0,0 +1,13 @@
+name: Link Checking
+
+on:
+  schedule:
+    - cron: 0 0 * * * # At 00:00 (UTC)
```

Comment:
Why run this on a timed schedule instead of in response to a branch push? It's not like things can change without a push.

Reply:
see https://github.com/tc39/proposals/pull/315#issuecomment-700347955, thanks

Reply:
@michaelficarra link validity can change at any time even if there's no pushes.

Reply:
@ljharb oh right, good point